tools-overview.md

```diff
@@ -0,0 +1,349 @@
+| [luis](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/MSBot/docs/add-services.md#connecting-to-luis-application)                 | connect to a LUIS application            |
+| [qna](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/MSBot/docs/add-services.md#connecting-to-qna-maker-knowledge-base)          | connect to QNA a service                 |
+
+See [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/MSBot) to learn more about the MSBot CLI.
+
+## Create and manage LUIS applications using LUIS CLI
+
+Included in the new tool set is a [LUIS extension](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/LUIS) which allows you to independently manage your LUIS resources. It is available as an npm module which you can download:
+
+```shell
+npm install -g luis-apis
+```
+The basic command usage for the LUIS tool from the CLI is:
+
+```shell
+luis <action> <resource> <args...>
+```
+To connect your bot to LUIS, you will need to create a **.luisrc** file. This is a configuration file which provisions your LUIS appID and password to the service endpoint when your application makes outbound calls. You can create this file by running **luis init** as follows:
+
+```shell
+luis init
+```
+You will be prompted in the terminal to enter your LUIS authoring key, region, and appID before the tool will generate the file.  
+
+Once this file is generated, your application will be able to consume the LUIS .json file (generated from LUDown) using the following command from the CLI.
+
+```shell
+luis import application --in luis-app.json | msbot connect luis --stdin
+```
+See [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/LUIS) to learn more about the LUIS CLI.
+
+## Create QnA Maker KB using QnA Maker CLI
+
+Included in the new tool set is a [QnA extension](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/QnAMaker) which allows you to independently manage your LUIS resources. It is available as an npm module which you can download.
+
+```shell
+npm install -g qnamaker
+```
+With the QnA maker tool, you can create, update, publish, delete, and train your knowledge base. You can use files generated via [ludown parse toqna](#generate-qna-knowledge-base) command to create/ replace a knowledge base.
+
+```shell
+qnamaker create --in qnaKB.json --msbot | msbot connect qna --stdin
+```
+
+See [here](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/QnAMaker) to learn more about the QnA Maker CLI.
+
+## Create dipsatch model using Dispatch CLI
+
+Dispatch is a tool to create and evaluate LUIS models used to dispatch intent across multiple bot modules such as LUIS models, QnA knowledge bases and others (added to dispatch as a file type).
```

Comment:
**The Dispatch tool is used to create, evaluate, and dispatch intent across multiple LUIS models and QnA knowledge bases** 

Not clear what the last part is trying to say.